src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ChromelessFormExtension.cs

```diff
@@ -14,7 +14,7 @@ public class ChromelessFormExtension : ITripleColonExtensionInfo
     {
         public string Name => "form";
         public bool SelfClosing => true;
-        public bool EndingTripleColons => false;
+        public bool EndingTripleColons {get; set;} = true;
```

Comment:
Is this supposed to be false?

Reply:
You are wonderful. Thanks for the catch :).